mantidimaging/gui/windows/live_viewer/live_view_widget.py

```diff
@@ -18,6 +23,10 @@ class LiveViewWidget(GraphicsLayoutWidget):
     @param parent: The parent widget
     """
     image: MIMiniImageView
+    image_shape: tuple[int, ...] | None = None
```

Comment:
`tuple[int, int]` is a bit more specific as it fixes the length to 2

Reply:
Ok, not a big issue.

One thing to try would be to put `assert len(image.shape) == 2` in `show_image()`.